Runtime/Properties/PropertyReflectionHelper.cs

```diff
@@ -17,9 +17,21 @@ namespace Innoactive.Creator.Core
     /// </summary>
     internal static class PropertyReflectionHelper
     {
-        /// <summary>
-        /// Extracts all <see cref="LockablePropertyData"/> from given Condition.
-        /// </summary>
+        public static List<LockablePropertyData> ExtractLockablesFromStep(IStepData data)
```

Comment:
So, no more documentation? ☹️

Reply:
I guess the name of the method is more then sufficient to explain what is happening :P